digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -907,7 +906,7 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
         .map(a => (roadLinkService.fetchVVHRoadlinks(Set(a._1)).headOption, a._2))
 
       existingLinksAndMeasures.foreach { asset =>
-        if (user.isAuthorizedToWriteInArea(maintenanceRoadService.getAssetArea(asset._1, asset._2), asset._1.get.administrativeClass))
+        if (!user.isAuthorizedToWriteInArea(maintenanceRoadService.getAssetArea(asset._1, asset._2), asset._1.get.administrativeClass))
```

Comment:
It's suppose to be `not (!)` ?

Reply:
Yes, '!' is correct. Noticed it as i was testing all the assets.